src/Parser/Rogue/Subtlety/Modules/Features/Abilities.js

```diff
@@ -20,16 +22,78 @@ class Abilities extends CoreAbilities {
     },
     {
       spell: SPELLS.SYMBOLS_OF_DEATH,
-      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
-      getCooldown: haste => 30, // TODO: Add T20 4Sets, reduce cd by 5 sec.
+      category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+      getCooldown: (haste, combatant) => (30 - (combatant.hasBuff(SPELLS.SUB_ROGUE_T20_2SET_BONUS.id) ? 5 : 0)), 
       recommendedCastEfficiency: 0.95,
+      importance: ISSUE_IMPORTANCE.MAJOR,
+      extraSuggestion: "This is the most important rotational ability, try to allways use it on cooldown.",      
     },
     {
       spell: SPELLS.SHADOW_DANCE,
-      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+      category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
       getCooldown: haste => 60, // TODO: Reduced by a passive.
       charges: 2,
-      recommendedCastEfficiency: 0.95,
+      recommendedCastEfficiency: 1,
+    },
```

Comment:
you have something with a recommendedCastEfficiency of 1, but no extraSuggestion to tell people how they should be using it if they don't follow this metric?

Reply:
This is old code, 1 is more correct then 0.95. Changed to noSuggestion, because the metric is not usable before cooldown reduction is implemented.

Reply:
Just so you know, the framework for cooldown reduction is handled for Cast Efficiency as of yesterday. The logic for the reduction still needs to be implemented on your end, if that's what you meant by it not being usable.

Reply:
And fwiw it doesn't have any ``noSuggestion`` added either

Reply:
Yeah I saw that you were working on it, but I will first need to implement Combo Point tracking so this has to be delayed =)